pkg/scheduler/api/job_info.go

```diff
@@ -485,7 +485,7 @@ func (ji *JobInfo) GetMinResources() *Resource {
 }
 
 func (ji *JobInfo) GetElasticResources() *Resource {
-	if ji.Allocated.LessEqualPartly(ji.GetMinResources(), Zero) {
+	if ji.Allocated.LessPartly(ji.GetMinResources(), Zero) {
```

Comment:
I'm not sure if there are other design considerations here, but if changed to `LessPartly`, the following place may assert exit.

https://github.com/volcano-sh/volcano/blob/d5bd18b7ad8f53bf29bf49cbdbebb6d35cd90bbe/pkg/scheduler/api/resource_info.go#L245-L247